config.json

```diff
@@ -8,7 +8,7 @@
     "representer": false,
     "analyzer": false
   },
-  "blurb": "TODO: add blurb",
+  "blurb": "Sequential commands automate tasks in Windows OS via .bat or .cmd files.",
```

Comment:
Usually this is a bit "promotional", where your selling the language a bit. The goal is for students to get a quick feel for the language and why they should be interested.

As an example, this is C#'s blurb:

> "C# is a modern, object-oriented language with lots of great features, such as type-inference and async/await. The tooling is excellent, and there is extensive, well-written documentation."

I'm happy for this to be improved later on of course.